src/services/sync/strategies/mixins/ClassImporter.js

```diff
@@ -56,6 +58,10 @@ module.exports = (superClass) => class ClassImporter extends superClass {
 			}
 			const existing = await this.findClass(query || options);
 			if (existing) {
+				if (onlyAddNew) {
+					options.teacherIds = Array.from(new Set((existing.teacherIds || []).concat(options.teacherIds)));
```

Comment:
Maybe for Array.from(new Set((existing.teacherIds || []).concat(options.teacherIds))) create a named helper to show why it is used? unique..?